server/src/main/java/org/spine3/server/event/EventDelivery.java

```diff
@@ -30,8 +30,8 @@
 import java.util.concurrent.Executor;
 
 /**
- * Base functionality for the routines delivering the {@link org.spine3.base.Event}s to event consumers,
- * such as {@link EventDispatcher}s or {@link EventSubscriber}s.
+ * Base functionality for the routines delivering the {@linkplain org.spine3.base.Event Events}
```

Comment:
It was `{@link org.spine3.base.Event}s`
Now is `{@linkplain org.spine3.base.Event Events}`
Please move `s` out of brackets if it is typo.